sky/packages/sky/lib/src/gestures/tap.dart

```diff
@@ -11,7 +11,9 @@ import 'events.dart';
 import 'pointer_router.dart';
 import 'recognizer.dart';
 
-typedef void GestureTapCallback();
+typedef void GestureTapCallback(ui.Point globalPosition);
+typedef void GestureTapDownCallback();
+typedef void GestureTapCancelCallback();
```

Comment:
I don't think we should pass the full event data in the tap callback. The whole point of the gesture recognisers is to abstract out the raw data and present a fully formed gesture. If you want the raw data, you should be at the level below this, either directly listening to the pointer events (as InkWell does), or as a GestureRecogniser, translating the raw data into gestures.